server/api/user/index.js

```diff
@@ -32,5 +32,6 @@ router.delete('/:id/project', auth.isAuthenticated(), controller.removeProject);
 router.put("/:id/addTech", auth.canEdit(), controller.addTech);
 router.put("/:id/removeTech", auth.canEdit(), controller.removeTech);
 router.post('/resetPassword', controller.resetPassword);
+router.put('/:id/removeUser',auth.isAuthenticated(),controller.destroy); 
```

Comment:
hehe, this will allow any logged in user to delete another user

Also, where is `controller.destroy` defined?